src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -51,6 +51,7 @@ public static partial class PlatformDetection
         public static bool IsNotArm64Process => !IsArm64Process;
         public static bool IsArmOrArm64Process => IsArmProcess || IsArm64Process;
         public static bool IsNotArmNorArm64Process => !IsArmOrArm64Process;
+        public static bool IsArmv6Process => (int)RuntimeInformation.ProcessArchitecture == 7; // Architecture.Armv6
```

Comment:
If the previous build failed on `RuntimeInformation.ProcessArchitecture == Architecture.Armv6` with ```/__w/1/s/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs(54,100): error CS0117: 'Architecture' does not contain a definition for 'Armv6' [/__w/1/s/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj]```,
I'm not expecting this to be true as then `RuntimeInformation.ProcessArchitecture` shouldn't find it either.

Reply:
The .csproj compiles against the net6.0 reference assemblies which didn't have the new Armv6 Architecture enum member.

The actual implementation at runtime does have it though and since enums are just integers at runtime we can compare against the numeric value.

Reply:
We can double check by running outerloop leg against this PR, but it should be possible for newer runtime to assign a higher value for which there is no enum member in library, e.g.:

```c#
var dayOfWeek = (System.DayOfWeek)42;
System.Console.WriteLine(dayOfWeek);
```
it can be cleaned up once https://github.com/dotnet/runtime/blob/d075b1f070cf255f60b6c986310796868f6aae66/global.json#L3 is bumped to 7 P1 etc.

Reply:
@akoeplinger beat me to it :D